.travis.yml

```diff
@@ -5,7 +5,7 @@ language: ruby
 cache: bundler
 sudo: false
 rvm:
-- 2.3.1
+- 2.5.1
```

Comment:
Should we do both a chef 12 and a chef 14 run?

Reply:
Why would we want to support an unsupported version of Chef with these new updates?
I am thinking no, but if you know a good reason then certainly.

Reply:
If our cookbook supports both chef 12 and 14, then it's not a major version number bump, and we make it easier on consumers who have to do both Chef upgrades and cookbook updates, as opposed to forcing a lock step of upgrade chef, then upgrade cookbook.

Reply:
I am not seeing how to do this, I believe the version is hardcoded in the [Gemfile](https://github.com/cerner/cerner_splunk/blob/stable/Gemfile#L10)

Reply:
Added the updates here: 312b003